apps/client/src/app/components/admin-jobs/admin-jobs.html

```diff
@@ -13,122 +13,163 @@
           </mat-select>
         </mat-form-field>
       </form>
-      <table class="gf-table w-100">
-        <thead>
-          <tr class="mat-header-row">
-            <th class="mat-header-cell px-1 py-2 text-right">#</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Type</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Symbol</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Data Source</th>
-            <th class="mat-header-cell px-1 py-2 text-right" i18n>Attempts</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Created</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Finished</th>
-            <th class="mat-header-cell px-1 py-2" i18n>Status</th>
-            <th class="mat-header-cell px-1 py-2">
+      <table class="gf-table w-100" mat-table [dataSource]="dataSource">
+        <ng-container matColumnDef="index">
+          <th *matHeaderCellDef class="px-1 py-2 text-right" mat-header-cell>
+            <ng-container>#</ng-container>
```

Comment:
When there is no `i18n` directive, you don't need `<ng-container>`. Can you remove it?